lib/pure/collections/sequtils.nim

```diff
@@ -47,6 +47,25 @@ proc concat*[T](seqs: varargs[seq[T]]): seq[T] =
       result[i] = itm
       inc(i)
 
+proc repeat*[T](s: seq[T], n: int): seq[T] =
+  ## Returns a new sequence with the items of `s` repeated `n` times.
+  ##
+  ## Example:
+  ##
+  ## .. code-block:
+  ##
+  ##   let
+  ##     s = @[1, 2, 3]
+  ##     total = s.repeat(3)
+  ##   assert total == @[1, 2, 3, 1, 2, 3, 1, 2, 3]
+  assert n is Natural, "Can't repeat sequence less than zero times"
```

Comment:
Sure. Sorry for my ignorance! I like it! Thats a reason why I wanted to use Nim :+1